.github/workflows/converter_tests.yaml

```
@@ -43,12 +43,12 @@ jobs:
       - uses: actions/setup-python@v6
         with:
           python-version: ${{ matrix.python-version }}
-          cache: 'pip'
-          cache-dependency-path: 'galaxy root/requirements.txt'
+      - name: Install uv
+        uses: astral-sh/setup-uv@v6
       - name: Move test data
         run: rsync -av --remove-source-files --exclude .git galaxy-test-data/ 'galaxy root/test-data/'
       - name: Install planemo
-        run: pip install planemo
+        run: uv pip install --system planemo
```

If we're already using uv, why not either use `uvx planemo` or `uv tool install planemo` ?

```suggestion
        run: uv tool install planemo
```